static/app/views/performance/transactionSummary/content.tsx

```diff
@@ -419,11 +421,13 @@ class SummaryContent extends React.Component<Props, State> {
               transactionName={transactionName}
               eventView={eventView}
             />
-            <StatusBreakdown
-              eventView={eventView}
-              organization={organization}
-              location={location}
-            />
+            {!isFrontendView && (
```

Comment:
Do I need to do anything smarter than this? Like actually inspect the table data and see if we get transaction.status === 'unknown' and then hide it?

Reply:
No, ideally we should never have async elements affecting layout (since it's visually noisy).